atom/renderer/atom_sandboxed_renderer_client.cc

```diff
@@ -158,6 +158,7 @@ void AtomSandboxedRendererClient::InitializeBindings(
   process.SetMethod("hang", AtomBindings::Hang);
   process.SetMethod("getHeapStatistics", &AtomBindings::GetHeapStatistics);
   process.SetMethod("getSystemMemoryInfo", &AtomBindings::GetSystemMemoryInfo);
+  process.SetMethod("getSystemMemoryInfo", &AtomBindings::GetProcessMemoryInfo);
```

Comment:
@codebytere Should this be `getProcessMemoryInfo` instead of `getSystemMemoryInfo` or are they simply aliases?